### DISCUSSION TITLE
How long do I have to wait for support to answer my reinstatement request?

### DISCUSSION THREAD

**Discussion by stil-blue** [community/community#111304] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My account @schlagmichdoch got flagged as spam by GitHub on 17th February. The process was silent without any notification. All repositories, PR, issues and comments are shadow hidden to the public and some issues and comments were even deleted by something called 'hubot'.

After realizing my account had been flagged, I created a reinstatement request ticket with the private GitHub Support one day later. I have been waiting 19 days and I have still not received a response or any notification at all.

This is a lot longer than what the info box currently says: „ Due to high support volume, it may take up to seven business days to hear back from our Support Engineers.“ and not acceptable.

This happened although I have public repository hosting libre software with 2.3k stars and clearly am not a bot. I’m very disappointed in the whole process and consider moving the whole project to another platform.

How long will I need to wait for a response?
Ticket #2603911

Thanks in advance!

**Comment by manarea** [comment]
I had the very same issue! My ticket has been waiting for a response for like +40 days and still nothing :(

**Comment by schlagmichdoch** [comment]
After 32 days my account was unflagged and is visible to the public again. There are still deleted issues and comments that need to be restored and I still do not have any information from GitHub about the reasoning but at least the account is back again. Going forward I will move to Codeberg soonish: https://codeberg.org/

**Comment by kiriclope** [comment]
I am experiencing the same arbitrary flagging, and am still waiting for a response after three weeks. https://codeberg.org/ is a great alternative and the community is quite responsive and supportive.

**Comment by OneOfTheMicahs** [comment]
My account was also flagged with absolutely no email or notification that it happened. I opened a community discussion and support ticket, but haven't heard from either of them!

**Reply by JackTNorris** [reply]
@OneOfTheMicahs any news?

**Comment by paulospx** [comment]
I can relate to your frustration, as I've encountered a similar issue with GitHub. My account was blocked a few months ago, and despite repeated inquiries, GitHub has not provided a clear reason for the suspension. I've wondered if it might be related to having a Ukrainian flag on my profile, but there's been no confirmation on that front. This lack of transparency and inability to back up or delete my account seems to be a violation of GDPR rights. Like you, I'm considering legal action as a last resort.

It's particularly challenging because this situation has significantly hindered my ability to work and access important materials. If you find a way to retrieve your data or if your community friends can offer any assistance, I would appreciate hearing about any solutions that might also apply to my situation.

Paulo

**Comment by B-Ilyas** [comment]
I have the same problem too. No email or notification to tell me I am flagged as "spammy". At least warn us !!

**Reply by B-Ilyas** [reply]
Ticket #2769876

**Reply by HamzaChbit** [reply]
now ?

**Comment by CoderSid063** [comment]
I also face the same issue that my account was marked as a spam , so i was unable to push my commits to the github , and i also raise a ticket for Reinstatement but still there is no response from the git support. already 20 days passed , I don't know when my account will be removed from spam .

**Reply by HamzaChbit** [reply]
Has your problem been solved? I am almost the same as you and support did not respond to me.

**Comment by Quy-Anh** [comment]
Facing the same issue right now, and from what I'm reading I'm afraid it'll take weeks as well before this gets fixed. GitHub really needs to fix this issue for the sake of people's work not being at stake.

**Comment by TahmidFarzan** [comment]
I have the same problem too. 
No answer coming from @github-staff.
My ticket : https://support.github.com/ticket/personal/0/2916054

**Comment by baharbaba** [comment]
I've been waiting for a response for 1 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.

**Reply by Quy-Anh** [reply]
Nearly 3 weeks by now and still no response or fix @github-staff. Can’t even do my work rn what’s with this stupid response time

**Comment by Quy-Anh** [comment]
Alright it has been over a month now @github-staff. I don't know what is taking so long, but for anyone having this problem, it may be good to consider stepping over to a different platform. Several times I have asked to fix this issue as my work is dependent on my repositories, but their supposed "7 business days" is actually 30+ days. I lost hope in this bs and I don't even know whether this will actually get solved. @github-staff @github-staff @github-staff

**Comment by cheesycod** [comment]
I have the same issue @github-staff

**Comment by Dave-Senn** [comment]
waiting since 13. nov 2024. what a shitshow

**Comment by Dave-Senn** [comment]
@github-staff yan you take a look at ticket #3094630 ?

**Comment by Abdullahdaro** [comment]
I have the same issue, @github-staff

**Comment by hklst4r** [comment]
I have the same issue. Would you mind please check support ticket #3244545 ? @github-staff

**Comment by kebufu** [comment]
![image](https://github.com/user-attachments/assets/6fb852a3-ca1c-4337-ba67-f996b2c80afe)
Github is a great open source platform 👍

**Reply by Abdullahdaro** [reply]
it has been since 20/01 and they dont answer at all 
bad support

**Reply by kebufu** [reply]
You never get any reply because you're BANNED

**Reply by Abdullahdaro** [reply]
how i will solve this problem then ????

**Reply by kebufu** [reply]
Create ticket with other account or STOP using github

**Reply by muzikbike** [reply]
absolutely shameful

**Comment by kebufu** [comment]
![image](https://github.com/user-attachments/assets/9415a0db-a62d-47f4-baf6-a76e32b8f372)
As you can see, github staff just doesn't reply your ticket, you may never get unban

**Comment by kebufu** [comment]
![image](https://github.com/user-attachments/assets/08065b0b-f132-412e-8d54-8b5f153d0e72)

Created a new ticket again, If still got closed agian, the best way is stop use github

**Comment by Pr0methean** [comment]
I have the same issue, but for an organization. My ticket is #3710400.
